modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -243,24 +258,58 @@ If successful, the call returns an object containing all current settings for th
 == XDCR Advanced Settings
 
 The advanced settings for XDCR are as follows.
-Note that the subset of advanced settings that can be configured by means of Couchbase Web Console is provided at xref:xdcr-reference:xdcr-advanced-settings.adoc[XDCR Advanced Settings].
+The subset of advanced settings that can be configured by means of Couchbase Web Console is provided at xref:xdcr-reference:xdcr-advanced-settings.adoc[XDCR Advanced Settings].
 
 .XDCR Advanced Settings
 [#table_xdcr_settings,cols="140,80,200"]
 |===
 | Parameter | Value | Description
 
 | `checkpointInterval`
-| Integer (60 to 14400).
-| Default: 600.
+| Integer (60 to 14400)
+| Default: 600
 The interval for checkpointing in seconds.
 
+This setting can be established and retrieved either for an individual replication or globally.
+
+| `colMappingRules`
+| JSON Object
+| Collection-related rules according to which explicit mapping or migration is defined for the replication.
+See xref:learn:clusters-and-availability/xdcr-with-scopes-and-collections.adoc#rules-for-explicit-mappings[Rules for Explicit Mappings] and xref:learn:clusters-and-availability/xdcr-with-scopes-and-collections.adoc#rules-for-migration[Rules for Migration].
+
+This setting can only be established for and retrieved from an individual replication: it cannot be established or retrieved as part of global settings.
+
+| `collectionsExplicitMapping`
+| Boolean (true or false)
+| Whether the replication uses explicit mapping: see xref:learn:clusters-and-availability/xdcr-with-scopes-and-collections.adoc#rules-for-explicit-mappings[Rules for Explicit Mappings].
+
+This setting can only be established for and retrieved from an individual replication: it cannot be established or retrieved as part of global settings.
+
+| `collectionsMigrationMode`
+| Boolean (true or false)
+| Whether the replication uses migration mode: see xref:learn:clusters-and-availability/xdcr-with-scopes-and-collections.adoc#migration[Migration].
+
+This setting can only be established for and retrieved from an individual replication: it cannot be established or retrieved as part of global settings.
+
+| `collectionsMirroringMode`
+| Boolean (true or false)
+| Whether the replication uses an implicit mapping: see xref:learn:clusters-and-availability/xdcr-with-scopes-and-collections.adoc#implicit-mapping[Implicit Mapping].
+
+This setting can only be established for and retrieved from an individual replication: it cannot be established or retrieved as part of global settings.
+
+| `collectionsOSOMode`
+| Boolean (true or false)
+|
```

Comment:
The description should say:
"Whether the replication optimizes performance by streaming mutations from source bucket that could be out of sequence number order"
Default is true

Reply:
I have added.

Reply:
Whether the replication optimizes performance; by streaming, from a source bucket, mutations that could be out of order, in terms of sequence-number.